__tests__/benchmarks/index.test.ts

```diff
@@ -275,27 +275,27 @@ const setup = async (scope = "tables") => {
     (acc, key) => {
       const collection = rxDb[key];
 
-      const get = async (record: $Record) => {
-        const item = await collection.findOne(record).exec();
+      const get = async (entity: $Record) => {
```

Comment:
Why is $Record still here?

Reply:
This is because we're testing against the NPM package at the time of the TinyBase integration, which was still using `$Record`.

I can alias these.